src/main/java/seedu/address/logic/orderer/FoodSelector.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.orderer;
```

Comment:
@samzx What's the reason for the refactor / new `FoodSelector` class? Reuse later on? Other than that, looks good!